### PR TITLE
SyncCalendars and One Callendar in two accounts [#342] [#334]

### PR DESCRIPTION
**Pull request by hekeemje**
https://user-images.githubusercontent.com/79774156/189955474-f5b6f9da-e148-4714-9261-5858ae1d7379.mp4

as you can see here if I do changes in my Google Calendar  - this event show in schedule when we create Meeting

https://user-images.githubusercontent.com/79774156/189957637-1e906d93-ec0b-45cb-9269-298789b7fcb2.mp4

here as you can see I add one Google callendar in two accounts

### CODE REVIEW

File: backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/CalendarsService.cs [hunk 1]
```diff
@@ -200,17 +206,20 @@ private async Task UpdateVisibleForTeamsTable(Calendar calendar, UserCalendarDto
 
         public async Task<bool> SyncChangesFromGoogleCalendar(string email)
         {
-            var calendar = await _context.Calendars.FirstOrDefaultAsync(x => x.ConnectedCalendar == email);
+            var calendars = await _context.Calendars.Where(x => x.ConnectedCalendar == email).ToListAsync();
 
-            if (calendar is null)
+            if (calendars is null)
```
Comment (Ymkavi) [inline backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/CalendarsService.cs]: ToListAsync on an empty sequence will return an empty list, not null

**Review by Ymkavi — CHANGES_REQUESTED** [review]

**Review by Ymkavi — CHANGES_REQUESTED** [review]
My friday event is not synced:
image
image

**Comment by hekeemje** [comment]
> My friday event is not synced: image image

on this issue already Yura works

**Review by Ymkavi — APPROVED** [review]

File: backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/CalendarsService.cs [hunk 2]
```diff
@@ -35,7 +35,7 @@ public async Task<bool> CreateGoogleCalendarConnection(TokenResultDto tokenResul
 
             var connectedEmail = response.Id;
 
-            if (await _context.Calendars.AnyAsync(el => el.ConnectedCalendar == connectedEmail))
+            if (await _context.Calendars.Where(el => el.ConnectedCalendar == connectedEmail).AnyAsync(el => el.UserId == currentUser.Id))
```
Comment (tatianahutii) [inline backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/CalendarsService.cs]: You can combine where and any

**Review by tatianahutii — CHANGES_REQUESTED** [review]

**Review by tatianahutii — APPROVED** [review]

**Review by apofis96 — APPROVED** [review]

**Pull request merged**